<launch>
    <node name="action_node_multi_server" pkg="ros_multithread_test" type="action_node_multi_server" output="screen"/>
</launch>